### ISSUE TITLE
snowflake_resource_monitor argument set_for_account cannot be found

### ISSUE THREAD

**Issue by eickeee** [chanzuckerberg/terraform-provider-snowflake#779]
**Provider Version**
v0.25.28.

**Terraform Version**
v1.0.11

**Describe the bug**

According to release https://github.com/chanzuckerberg/terraform-provider-snowflake/releases/tag/v0.25.28 
the following commit was merged: https://github.com/chanzuckerberg/terraform-provider-snowflake/commit/2613aa31da958e3557849e0615067c649c704110 from @gouline 

However, when using the latest release of the provider, terraform says `An argument named "set_for_account" is not expected here.`

Additionally, the documentation (https://registry.terraform.io/providers/chanzuckerberg/snowflake/latest/docs/resources/resource_monitor) also does not state the parameter even though it was added to `docs/resources/resource_monitor.md` in the commit.

Example code:
```
resource "snowflake_resource_monitor" "resource_monitor_account_level" {
  name                       = "resource_monitor_account_level"
  credit_quota               = 100
  set_for_account = true
}
```
![image](https://user-images.githubusercontent.com/5420972/144829431-affb3eba-e9af-4c2f-8c25-50888f733cec.png)

**Expected behavior**
It can be found and works 

Could someone please verify it's a bug with the release/provider or if I have a mistake on my side - thanks 🙏

**Comment by gouline** [comment]
Release [v0.25.28](https://github.com/chanzuckerberg/terraform-provider-snowflake/releases/tag/v0.25.28) is commit 5fefd92, whereas my commit 2613aa3 was after that. I'd expect it to be released in v0.25.29.

**Comment by eickeee** [comment]
Sorry, misread👇 in a way that the release contains of the commits - my mistake, thanks!
![image](https://user-images.githubusercontent.com/5420972/144843389-75d1fe6e-21c6-4bea-962c-f0faef5a9079.png)
